src/status_im2/contexts/communities/actions/addresses_for_permissions/view.cljs

```diff
@@ -0,0 +1,54 @@
+(ns status-im2.contexts.communities.actions.addresses-for-permissions.view
+  (:require [quo.core :as quo]
+            [react-native.core :as rn]
+            [status-im2.contexts.communities.actions.addresses-for-permissions.style :as style]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
+
+(defn- account-item
+  [item]
+  [rn/view
+   {:style style/account-item-container}
+   [quo/account-avatar (assoc item :size 32)]
+   [rn/view
+    [quo/text
+     {:size   :paragraph-1
+      :weight :semi-bold} (:name item)]
+    [quo/address-text
+     {:address (:address item)
+      :format  :short}]]])
```

Comment:
Do we need this?

We have the `[quo.components.list-items.account.view :as account-view]` which can be passed as:

```
(defn- render-item
  [item]
  [account-view/view
   {:account-props item
    :emoji         (:emoji item)}])
```
Please let me know if Im wrong so we can follow the same pattern.

Reply:
Continue using the existing `account` component in your implementation, that's the correct component for your screen as per Figma. However, for this specific screen, the component we should use is [`account-permissions`](https://github.com/status-im/status-mobile/issues/17978) which is not implemented yet. I've started the implementation and will finish it once [this PR](https://github.com/status-im/status-mobile/pull/18164) is merged. Later on, we can replace this custom component with the quo component in a subsequent PR.

It's more efficient to handle the quo implementation in separate PRs. Combining these changes may slow us down, as they necessitate testing and design review.